src/components/Card/Chart/PeriodPostLikeCard.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from "react";
+
+import asyncGetPostLikeList from "../../../api/keyword/asyncGetPostLikeList";
+import { getCursorDate } from "../../../utils/date";
+import LineChart from "../../Chart/LineChart";
+import PeriodPagination from "../../Pagination/PeriodPagination";
+import { keepPreviousData, useQuery } from "@tanstack/react-query";
+import PropTypes from "prop-types";
+
+const PeriodPostLikeCard = ({ keywordId }) => {
+  const [cursorId, setCursorId] = useState(() => getCursorDate());
+
+  const {
+    data: chartData,
+    isPlaceholderData,
+    isError,
+  } = useQuery({
+    queryKey: ["postLike", keywordId, cursorId],
+    queryFn: () => asyncGetPostLikeList(keywordId, cursorId),
+    select: (data) => ({ ...data, items: data.postLikeList }),
+    placeholderData: keepPreviousData,
+  });
+
+  if (chartData === undefined) {
+    return null;
+  }
+
+  if (isError || chartData?.message?.includes("Error occured")) {
+    return (
+      <div className="w-1/2 h-full p-10 border-2 rounded-md flex justify-center items-center">
+        주간 좋아요 수 차트를 불러오는 데 실패했습니다.
```

Comment:
```suggestion
        주간 공감 수 차트를 불러오는 데 실패했습니다.
```
용어를 '공감 수'로 맞춰보면 어때요?
의미상 '좋아요'와 다를 바가 없지만, 네이버 블로그에서 공식적으로 사용하는 용어와 맞추는게 정확하겠다고 생각했어요! cc. @SsongQ-92 
<img width="300" alt="Screenshot 2024-11-15 at 4 58 19 AM" src="https://github.com/user-attachments/assets/36e9cc66-7a30-4352-b853-3e525444dbe5">

Reply:
네네 좋아요! 용어를 통일시키는게 사용자 측면에서 생각해보면 대시보드 화면을 이해하기 더 쉬울 것 같네요!
@SsongQ-92 해당 부분에 대해서 어떻게 생각하실까요?!